src/Presence/kahoot.json

```
@@ -94,5 +94,97 @@
     "kahoot.of": {
         "message": "{0} of {1}",
         "description": "Used in the context of questions (i.e 1 of 3). {0} is the current question and {1} is the total amount of questions"
+    },
+    "kahoot.onPodium": {
+        "message": "On the Podium",
+        "description": "Used in the context of a player/team being withing the top 3"
+    },
+    "kahoot.pollAnswer": {
+        "message": "Answered Poll",
+        "description": "Used when a player is viewing the results of a poll"
+    },
+    "kahoot.drumRoll": {
+        "message": "Waiting for final ranking...",
+        "description": "Used when a player is waiting to receive their final ranking"
+    },
+    "kahoot.position": {
+        "message": "Position: {0}",
+        "description": "Used to display the position (ranking) of a player. For example, 'Position: 2'"
+    },
+    "kahoot.teamTalk": {
+        "message": "Discussing with team",
+        "description": "Used when a player is talking to their team during team talk time"
+    },
+    "kahoot.gameSummary": {
+        "message": "Looking at game summary",
+        "description": "Used when the host is viewing the summary of a game"
+    },
+    "kahoot.login": {
+        "message": "Logging in",
+        "description": "Used when a user is logging in"
+    },
+    "kahoot.createHome": {
+        "message": "Viewing the \"Create home\" page",
```

to be fair the page is called home on the website, don't know where you got this from. It could be changed to this which sounds more natural
```suggestion
        "message": "Viewing the home page",
```

---

The reason I added "Create" to the string is that it's the homepage for specifically `create.kahoot.it`, to avoid ambiguity.

---

You could provide info about the url elsewhere maybe? Not sure if they have different icons but maybe through a state or small image text?

---

They have the same icon, but I'll consider some kind of small image alternative.